src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -857,7 +900,7 @@ void EnIk_PostLimbDraw3(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Ve
 
     OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_ik_inFight.c", 1201);
 
-    if (this->unk_2FB & 1) {
+    if (this->armorStatusFlag & 1) {
```

Comment:
```suggestion
    if (this->armorStatusFlag & ARMOR_BROKEN) {
```
same with the few `drawArmorFlag & 1`